code/__DEFINES/reagents.dm

```diff
@@ -36,7 +36,10 @@
 #define CLEAR_REAGENTS	4	// all reagents were cleared
 
 #define MIMEDRINK_SILENCE_DURATION 30  //ends up being 60 seconds given 1 tick every 2 seconds
-#define THRESHOLD_UNHUSK 50 //Health treshold for synthflesh and rezadone to unhusk someone
+///Health threshold for synthflesh and rezadone to unhusk someone
+#define THRESHOLD_UNHUSK 50
```

Comment:
Just some quick bikeshedding. This is a bad variable name IMO, `UNHUSK_DAMAGE_THRESHOLD` is probably better.

Reply:
Agreed. Changed.